packages/e2e-tests/specs/editor/various/sidebar-permalink.test.js

```diff
@@ -5,17 +5,17 @@ import {
 	activatePlugin,
 	createNewPost,
 	deactivatePlugin,
-	findSidebarPanelWithTitle,
 	publishPost,
 } from '@wordpress/e2e-test-utils';
 
-const permalinkPanelXPath = `//div[contains(@class, "edit-post-sidebar")]//button[contains(@class, "components-panel__body-toggle") and contains(text(),"Permalink")]`;
+// TODO: Use a more accessible selector.
```

Comment:
I'll fix this in a follow up PR. It points to how we need to set proper `aria-label`s on these buttons. Right now screen readers read them as _Public_, _June 27_, _example.com/page_, _Default template_ which I can't imagine is very useful. It should be something like _Select visibility: Public_, _Change publish date: June 27_, _Change URL: example.com/page_, _Change template: Default template_.

Why a separate PR and not now? Because the issue already exists in `trunk` and I want proper a11y / copy feedback.

Reply:
We can also migrate the e2e tests to Playwright while we are at it 😄 The locator works great with labels.